aptible/resource_app.go

```diff
@@ -322,7 +322,7 @@ func resourceAppCreate(d *schema.ResourceData, meta interface{}) error {
 	// which I'm not prepared to do quite yet. So instead we're handling scaling after deployment, rather than
 	// at the time of deployment.
 	// TODO: We can check for services scaled to 1 GB/1 container before scaling.
-	err = scaleServices(d, meta)
+	err = scaleServices(context.Background(), d, meta)
```

Comment:
Might be an improvement to get an API context at the beginning of this method and pass that to scaleServices and updateServices instead of passing `context.Background()` to both and then converting it. Then again, I don't know that we're actually using the context for anything fancy so it might not matter, so feel free to ignore this comment if so.